packages/fluentui/react-builder/tsconfig.json

```diff
@@ -2,6 +2,7 @@
   "extends": "../../../scripts/typescript/tsconfig.fluentui",
   "compilerOptions": {
     "module": "esnext",
+    "moduleResolution": "node",
```

Comment:
Just out of curiosity, why was it needed?

Reply:
There were some issues with the editor keep saying `@material-ui/core` is not defined. Even though it was in package.json and added via yarn. The build was also failing because of this. This is fixing the problem.

Reply:
What about `declare module '@material-ui/core' `? Would it be better solution than changing the module resolution @layershifter ?